src/diff/index.js

```diff
@@ -215,6 +225,11 @@ export function diff(
 
 		if ((tmp = options.diffed)) tmp(newVNode);
 	} catch (e) {
+		// Before bailing out, mark the current VNode with the DOM element and hydration state.
+		// We can use this information if we return here to render later on.
+		newVNode._dom = oldDom;
```

Comment:
Can be reduced to:

```suggestion
		oldVNode._hydrateDom = newVNode._dom = oldDom;
```

This removes the need for the next line (multi-line suggestions aren't available)